src/ontology/mondo-edit.obo

```diff
@@ -340013,8 +340026,7 @@ subset: orphanet_rare {source="Orphanet:643549"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "HAFOUS" EXACT ABBREVIATION [OMIM:616863]
-synonym: "Hao-Fountain syndrome" EXACT CLINGEN_LABEL []
-synonym: "USP7-Related Diseases" EXACT [NORD:1917]
+synonym: "Hao-Fountain syndrome" EXACT []
```

Comment:
since this synonym is the same as the term label, and since there is no additional reason to have this synonym (eg to indicate the ClinGen Label), this synonym can be deleted.

Reply:
Thanks, fixed now.